src/AppBundle/Entity/Post.php

```diff
@@ -95,7 +95,7 @@ class Post
      *      mappedBy="post",
      *      orphanRemoval=true
      * )
-     * @ORM\OrderBy({"publishedAt" = "DESC"})
+     * @ORM\OrderBy({"publishedAt": "DESC"})
```

Comment:
@javiereguiluz, we already use the `:` in routes: https://github.com/symfony/symfony-demo/blob/master/src/AppBundle/Controller/BlogController.php#L38